modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -61,6 +71,78 @@ async function getSiteHomePageScreenshot(
 	return screenshot;
 }
 
+test(
+	'make sure we do not export-import wikiNodes if they are not selected in the export configuration screen',
+	{tag: '@LPS-195766'},
+	async ({
+		exportImportPage,
+		page,
+		pageEditorPage,
+		pageTemplatesPage,
+		site,
+		wikiPage,
+	}) => {
+		await wikiPage.goto(site.friendlyUrlPath);
+
+		await wikiPage.createNewWikiNode('Wiki Node Title');
+
+		await pageTemplatesPage.goto(site.friendlyUrlPath);
+
+		// Create page template collection
+
+		const pageTemplateCollectionName = getRandomString();
+
+		await pageTemplatesPage.addPageTemplateCollection(
+			pageTemplateCollectionName
+		);
+
+		await expect(
+			page.getByRole('menuitem', {
+				exact: true,
+				name: pageTemplateCollectionName,
+			})
+		).toBeVisible();
+
+		// Create widget page template
+
+		const pageTemplateName = getRandomString();
+
+		await pageTemplatesPage.addWidgetPageTemplate(pageTemplateName);
+
+		await pageTemplatesPage.page.getByLabel('Add', {exact: true}).click();
+
+		await pageEditorPage.addWidgetToWidgetPageTemplate(
+			'Content Management',
+			'Web Content Display'
+		);
+
+		await wikiPage.goto(site.friendlyUrlPath);
+
+		await exportImportPage.goToExport();
+
+		const exportName = 'MyExport-' + getRandomString();
+
+		await exportImportPage.createNewExportProcess(exportName);
+
+		await expect(
+			exportImportPage.page
+				.getByText(exportName)
+				.locator('../..')
+				.getByText('Successful')
+		).toBeVisible();
+
+		const exportFilePath =
+			await exportImportPage.downloadExportProcess(exportName);
+
+		await exportImportPage.goToImport();
+
+		await exportImportPage.checkItemInNewlyCreatedImportProcess(
+			exportFilePath,
+			'Wiki'
+		);
+	}
+);
```

Comment:
How do we handle what is created in the test elements? Maybe it's possible to remove the data using the `apiHelpers.data.push({id: ${element}.id, type: ' ${elementType}'});`?

Reply:
I didn't notice the isolated test is being used here, so there's no need to remove anything